### DISCUSSION TITLE
Field to hold Equipment Information

### DISCUSSION THREAD

**Discussion by M0THL** [magicbug/Cloudlog#1018] [Ideas / Feature Requests]
Thanks Peter for the work put into this project.

It'd be great if it were possible to put some equipment notes against a QSO.

E.G. Sometimes I use Rig 1 and Antenna 1, but sometimes I use Rig 1 and Antenna 2, othertimes Rig 2 and Antenna 1 etc. All at the same station. I'd like to note which equipment was used for a specific QSO.

Could MY_RIG/COL_MY_RIG be used?

**Comment by magicbug** [comment]
Yep this seems a reasonable request, I was going to suggest putting this in "Station Locations" but it might be wise to also put it in "Station" tab as well, its getting busy in there so might need a sub tab to split things up a bit.

**Comment by magicbug** [comment]
@M0THL Just to say this has been added to the roadmap https://github.com/magicbug/Cloudlog/projects/8
